static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/control.js

```diff
@@ -1,17 +1,67 @@
 // @flow
-import type { SignInGateVariant } from '../types';
-import { componentName } from '../component';
+import mediator from 'lib/mediator';
+import type { CurrentABTest, SignInGateVariant } from '../types';
+import { component, componentName } from '../component';
 import {
     hasUserDismissedGate,
     isNPageOrHigherPageView,
     isLoggedIn,
     isInvalidArticleType,
     isInvalidSection,
+    addOpinionBgColour,
+    addClickHandler,
+    setUserDismissedGate,
+    showGate,
 } from '../helper';
 
 // define the variant name here
 const variant = 'control';
 
+// add the html template as the return of the function below
+// signInUrl - parameter which holds the link to the sign in/register page with the tracking parameters added
+// guUrl - url of the STAGE frontend site, e.g. in DEV stage it would be https://m.thegulocal.com,
+//         and for PROD it would be https://theguardian.com
+const htmlTemplate: ({
```

Comment:
I think it would be good to refactor out these html templates and click handlers to another location, in this test they are all using the same design, and it is likely that this same design will be used for the control for the next test. By separating display logic from designs it makes it a lot easier to reuse or modify the designs separately : )